targets/targets.json

```diff
@@ -243,6 +243,17 @@
         "features": ["LWIP"],
         "device_name": "LPC1768"
     },
+    "LPC1769": {
+        "inherits": ["LPCTarget"],
```

Comment:
should this inherit from lpc1768 ? no extra label on the line 249 needed then? 

I believe that is what @chrissnow was suggesting earlier?